### PR TITLE
Preference Defaults and Keybinding support/defaults

### PR DESCRIPTION
**Pull request by MonaraMir**
Sets default FPS to 40 for new players instead of 0(server default, which is 20 currently)
Creates/Sets default rest keybinding to V
Sets default Move upwards keybinding to shiftC
Sets default Move downwards keybinding to ctrlC
Creates/Sets default subtler anti-ghost keybinding to 7

### CODE REVIEW

File: code/modules/keybindings/keybind/living.dm [hunk 1]
```diff
@@ -11,6 +11,12 @@
 	full_name = "Resist"
 	description = "Break free of your current state. Handcuffed? on fire? Resist!"
 
+/datum/keybinding/living/rest
+	hotkey_keys = list("V")
+	name = "rest"
+	full_name = "Rest"
+	description = "Toggle between lying down or standing up."
+
```
Comment (silicons) [inline code/modules/keybindings/keybind/living.dm:19]: need /down() behavior

File: code/modules/keybindings/keybind/mob.dm [hunk 2]
```diff
@@ -184,6 +184,12 @@
 	full_name = "Subtle"
 	description = "Does a subtle emote."
 
+/datum/keybinding/mob/subtler
+	hotkey_keys = list("7")
+	name = "Subtler Anti Ghost"
+	full_name = "Subtler Anti-Ghost"
+	description = "Does a subtle emote that's invisible to ghosts."
+
```
Comment (silicons) [inline code/modules/keybindings/keybind/mob.dm:192]: need /down behavior

File: code/modules/keybindings/keybind/living.dm (hunk 1 above)
Comment (MonaraMir) [reply code/modules/keybindings/keybind/living.dm:24]: Make sure I called the right proc on this one.

File: code/modules/keybindings/keybind/movement.dm
Comment (TheLordME) [inline code/modules/keybindings/keybind/movement.dm:1]: bay uses , and . for this

Comment (MonaraMir) [reply code/modules/keybindings/keybind/movement.dm:1]: I feel like ctrl/shift C is better for fighter combat, which is where it'll likely have the highest need for it to be accessible without moving the hand across the keyboard.

Comment (TheLordME) [reply code/modules/keybindings/keybind/movement.dm:1]: fair enough, just thought you'd like to know

**Review by TheLordME — APPROVED** [review]

**Pull request merged**